src/set_theory/zfc.lean

```diff
@@ -707,15 +724,17 @@ Class.iota_val _ _ (λz, by simp; exact
 variables (x : Set.{u}) (h : ∅ ∉ x)
 
 /-- A choice function on the set of nonempty sets `x` -/
-noncomputable def choice : Set := @map (λy, classical.epsilon (λz, z ∈ y)) (classical.all_definable _) x
+noncomputable def choice : Set :=
+@map (λy, classical.epsilon (λz, z ∈ y)) (classical.all_definable _) x
 
 include h
 theorem choice_mem_aux (y : Set.{u}) (yx : y ∈ x) : classical.epsilon (λz:Set.{u}, z ∈ y) ∈ y :=
 @classical.epsilon_spec _ (λz:Set.{u}, z ∈ y) $ classical.by_contradiction $ λn, h $
 by rwa ←((eq_empty y).2 $ λz zx, n ⟨z, zx⟩)
 
 theorem choice_is_func : is_func x (Union x) (choice x) :=
-(@map_is_func _ (classical.all_definable _) _ _).2 $ λy yx, by simp; exact ⟨y, yx, choice_mem_aux x h y yx⟩
+(@map_is_func _ (classical.all_definable _) _ _).2 $
+λy yx, by simp; exact ⟨y, yx, choice_mem_aux x h y yx⟩
```

Comment:
The second line should be indented because it is in brackets

Reply:
```suggestion
  λy yx, by simp; exact ⟨y, yx, choice_mem_aux x h y yx⟩
```